server/api_test.go

```diff
@@ -43,6 +43,12 @@ func TestParseSessionRequest(t *testing.T) {
 			res.SessionRequest().Disclosure().Disclose[0][0][0].Type.String())
 	})
 
+	t.Run("correct default validity", func(t *testing.T) {
+		res, err := ParseSessionRequest(requestJson)
+		require.NoError(t, err)
+		require.Equal(t, 120, res.Base().ResultJwtValidity)
```

Comment:
Here and elsewhere across this PR it would be nice if the magic number `120` would be replaced by the new constant `irma.DefaultJwtValidity`.

Reply:
I changed it here now but couldn't find more occurrences that I hadn't yet replaced earlier.